config/v1/types_infrastructure.go

```diff
@@ -1082,9 +1082,39 @@ type VSpherePlatformStatus struct {
 	LoadBalancer *VSpherePlatformLoadBalancer `json:"loadBalancer,omitempty"`
 }
 
+// IBMCloudServiceEndpoint stores the configuration of a custom url to
+// override existing defaults of IBM Cloud Services.
+type IBMCloudServiceEndpoint struct {
+	// Name is the name of the IBM Cloud service.
+	// examples of these services are:
+	//   IAM=https://cloud.ibm.com/apidocs/iam-identity-token-api
```

Comment:
I'll see if I can clear that up. Is there an example of using prose I can compare with?

Reply:
Well, what I think you might want to say is:
```
For example, the `IAM` service would normally have the URL ...
Whereas the `Foo` service would normally have the URL ...
```

Reply:
Perhaps something like this would be better:
```
       // For example, the IBM Cloud Private IAM service could be configured with the
        // service `name` of `iam` and `url` of `https://private.iam.cloud.ibm.com`
        // Whereas the IBM Cloud Private VPC service for US South (Dallas) could be configured
        // with the service `name` of `vpc` and `url` of `https://us.south.private.iaas.cloud.ibm.com`
```

Reply:
I updated the example with pascalcase